src/documentation/user_guides/malloy_by_example.malloynb

```diff
@@ -244,19 +230,17 @@ run: airports3 -> {
 
 ## Refining a Source-Level Query
 
-The refinement gesture `{}` extends an existing object, creating a new version with added properties
+The query refinement gesture `+ {}` extends an existing object, creating a new version with added properties
```

Comment:
I'm okay with saying "query refinement," but I don't think it's necessary. _There is no other kind of refinement anymore_. The old "source refinement" is now called "source extension" or just "extension," so "refinement" by itself is unambiguous.